plat/rpi/rpi5/include/rpi5_smc.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef PLAT_RPI_RPI5_INCLUDE_RPI5_SMC_H_
+#define PLAT_RPI_RPI5_INCLUDE_RPI5_SMC_H_
+
+#include <lib/smccc.h>
+
+/*
+ * Process scmi message pending in the SCMI message
+ * shared memory buffer.
+ */
+#define RPI5_SIP_SMC_SCMI 0x82000002
```

Comment:
Isn't there a generic SMC id for this type of call? I believe I saw patch series for linux that added SMC mailbox support...

Reply:
I couldn't find any of this in arm-tf. Digging into similar solutions from IMX and ST shows that they both define smc id.
As for the smc-mailboxes in the linux kernel - if you are talking about the following patch series:
https://lore.kernel.org/linux-arm-kernel/AM0PR04MB448121C973D3A62C4BD00722887B0@AM0PR04MB4481.eurprd04.prod.outlook.com/T/
Then, according to the bindings, it expects scmi-id to be retrieved from the device tree.